### DISCUSSION TITLE
Windows user - how to update to never version ?

### DISCUSSION THREAD

**Discussion by jomardyan** [open-webui/open-webui#7417] [Q&A]
Windows user - how to update to never version ? PIP method doesnt work 

![image](https://github.com/user-attachments/assets/9abc144c-eabc-4377-8bbe-a42d401a72fe)

**Comment by CSAnetGmbH** [comment]
Hello , did you use Docker ?
When yes, switch to commandline and put : 

docker run --rm --volume /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --run-once open-webui 

in there !

In the last part of the command, replace open-webui with your container name if it is different.

**Reply by jomardyan** [reply]
Using flat windows installation via PIP, do not using docker.

**Reply by CSAnetGmbH** [reply]
A ok, so I cant help you. Sorry, did not use it in Windows.
